src/index.html

```diff
@@ -1,12 +1,404 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" />
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo" />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link"> Samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link"> Smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops_and_computers" class="nav__link" data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link"> Gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link"> Tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link"> Photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div
+        class="card"
+        data-qa="card"
+       >
+         <img
+           src="./images/imac.jpeg"
+           alt="iMac"
+           class="card__img"
+         >
+
+         <h1 class="card__name">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+         </h1>
+
+         <span class="card__code">
```

Comment:
```suggestion
         <p class="card__code">
```
use `p` tag for text